charts/seed-controlplane/charts/kube-controller-manager/templates/kube-controller-manager.yaml

```diff
@@ -46,6 +46,9 @@ spec:
         networking.gardener.cloud/to-dns: allowed
```

Comment:
As you are already on it, can you split up the manifests in this file into multiple files?